src/kdl/include/kdl_codyco/undirectedtree.hpp

```diff
@@ -284,7 +284,7 @@ namespace CoDyCo
     public:
 
         UndirectedTreeJunction() {};
-        UndirectedTreeJunction(const std::string & name, const Joint & joint_in, const Frame & f_tip_in, const int q_nr_in = -1, const int body_part_in = -1, const int body_part_q_nr_in=-1):
+       	UndirectedTreeJunction(const std::string & name, const Joint & joint_in, const Frame & f_tip_in, const int q_nr_in = -1):
```

Comment:
You inserted a `tab` here. Tabs are tricky because they are visualized differently depending on the editor (they can be render as 2/4/8 spaces). For this reason I suggest to configure the editor to always emit spaces when you hit "Tab".

To modify this in KDevelop, go to : 
Settings -> Configure KDevelop -> Editor -> Editing -> Indentation

See https://opensourcehacker.com/2012/05/13/never-use-hard-tabs/